addon/components/rdf-input-fields/numerical-input.hbs

```diff
@@ -8,17 +8,30 @@
     {{@field.label}}
   </AuLabel>
 {{/unless}}
-<AuInput
-  @error={{this.hasErrors}}
-  @disabled={{@show}}
-  @width={{if @inTable "block"}}
-  @warning={{this.hasWarnings}}
-  id={{this.inputId}}
-  lang="nl-BE"
-  type="number"
-  value={{this.value}}
-  {{on "change" this.updateValue}}
-/>
+
+<div class="center-container">
+  <AuInput
+    class={{if @field.options.allowClear "number-input-no-spin"}}
+    @error={{this.hasErrors}}
+    @disabled={{@show}}
+    @width={{if @inTable "block"}}
+    @warning={{this.hasWarnings}}
+    id={{this.inputId}}
+    lang="nl-BE"
+    type="number"
+    value={{this.value}}
+    {{on "change" this.updateValue}}
+  />
+  {{#if @field.options.allowClear}}
+    <AuIcon
```

Comment:
Is this button needed? Wouldn't clearing the field like any other input field be more intuitive? I would also find it strange to only see this clear button for numerical inputs, but non of the other fields.

Reply:
Should we just add a quick return here https://github.com/lblod/ember-submission-form-fields/blob/master/addon/components/rdf-input-fields/simple-value-input-field.js#L50 I would look something like this

```js
get CanClearInputField() {
   return ... <LOGIC>
}

updateValue(value) {
    if(!value && this.CanClearInputField) {
        updateSimpleFormValue(this.storeOptions, null, this.nodeValue);
        super.updateValidations();
        
        return;
     }
}
```


All inputs that are extending from `SimpleValueInputFieldComponent` will have the option to clear.
Just a thought :)

Reply:
This seemed like the easier approach, but it might indeed be a bit confusing, will try your aproach.